README.md

```diff
@@ -118,7 +135,7 @@ To avoid the deprecation warning from the use of secrets in `ejson-rails` once y
 gem 'ejson-rails', require: 'ejson/rails/skip_secrets'
 ```
 
-This will no longer merge secrets from JSON in `Rails.application.secrets`. This will be the default in the next major version.
+This will no longer merge secrets from JSON into `Rails.application.secrets`. This will be the default in the next major version.
```

Comment:
```suggestion
With this require, ejson-rails will no longer merge secrets from JSON into `Rails.application.secrets`. This will be the default in the next major version.
```